app/assets/javascripts/guiders.js

```diff
@@ -0,0 +1,42 @@
+$(document).on('turbolinks:load', function() {
+  $('.js-go-link').on('click', function() {
```

Comment:
You can use `$(document).on(".js-go-link", "click", fn)` sort of style instead of waiting for `turbolinks:load`. In my experience this produces much more robust javascript.